frontend/src/components/auth/SignInForm.tsx

```diff
@@ -174,16 +172,8 @@ const SignInForm = ({ redirectPath }: { redirectPath: string }) => {
             })}
           >
             <TextInput
-              label={
-                config.get("ldap.enabled")
-                  ? t("signup.input.username")
-                  : t("signin.input.email-or-username")
-              }
-              placeholder={
-                config.get("ldap.enabled")
-                  ? t("signup.input.username.placeholder")
-                  : t("signin.input.email-or-username.placeholder")
-              }
+              label={t("signin.input.email-or-username")}
```

Comment:
Because of `const ldapUsername = dto.username || dto.email;` in `backend/src/auth/auth.service.ts` the user can't sign in with email, just with an username that looks like an email. Shouldn't we display the label "Username" instead of "Email or username" in that case?

Reply:
I have not had time to test the admin group login yet, that was regiolisas. The dev branch made it appear that a non signed in user could upload i had to revert back to the main branch. i dont have a test system only production. :(

Reply:
> Because of `const ldapUsername = dto.username || dto.email;` in `backend/src/auth/auth.service.ts` the user can't sign in with email, just with an username that looks like an email. Shouldn't we display the label "Username" instead of "Email or username" in that case?

Well, good question. I think neither of the proposed solutions would be adequate, as we don't always know when someone uses LDAP if they are expecting to log in with a username or an email (like in the case of @Esperenza84, for example).

When looking at how GitLab handles this, they’ve taken a very clean approach by explicitly separating both types of login. Maybe that could be an applicable solution here as well?
![image](https://github.com/user-attachments/assets/5c23c663-f74a-499f-b90f-4b678eefd541)

Reply:
Oh, I mistakenly believed that signing in “normally” (without LDAP) is disabled when LDAP is enabled. However, I just realized that this isn’t the case. In such a scenario, it would be logical to display the “email or username” label.

The implementation of GitLab is indeed clean, but I believe we could create a separate pull request in the future if this feature is requested.

What do think?

Reply:
Yes absolutely, this seems to be out of the scope of this PR :)